ui/ember-cli-build.js

```diff
@@ -33,9 +18,20 @@ module.exports = function(defaults) {
       plugins: ['transform-object-rest-spread'],
     },
     'ember-cli-babel': {
-      includePolyfill: true,
+      includePolyfill: isProd,
```

Comment:
This may need to be `isProd || isTest` so that IE testing works when we get there.